Client/Frontend/Browser/BrowserLocationView.swift

```diff
@@ -131,7 +134,8 @@ class BrowserLocationView : UIView, ToolbarTextFieldDelegate {
         readerModeButton.addGestureRecognizer(UILongPressGestureRecognizer(target: self, action: "SELlongPressReaderModeButton:"))
         addSubview(readerModeButton)
         readerModeButton.isAccessibilityElement = true
-        readerModeButton.accessibilityLabel = NSLocalizedString("Reader Mode", comment: "Accessibility label for the reader mode button")
+        readerModeButton.accessibilityLabel = NSLocalizedString("Reader View", comment: "Accessibility label for the Reader View button")
```

Comment:
Note that this change makes the reader tests fail, so please update them accordingly (I recommend re-running all tests whenever changing any accessibility labels since they're so tightly coupled to KIF).